develop/app/Database/Seeds/Product.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Database\Seeds;
+
+use CodeIgniter\Database\Seeder;
+use Faker\Factory;
+
+class Product extends Seeder
+{
+    public function run()
+    {
+        $faker = Factory::create();
+        $now   = date("Y-m-d H:i:s");
+
+        for ($i = 0; $i < 50; $i++) {
```

Comment:
The product has fifty data is too much, It may increase the db server load and it's meaningless.